README.md

```diff
@@ -87,13 +85,25 @@ Flags:
       --parse_metric_data bool                   When true, core files will be parsed and non-relevant data will be removed prior to upload. (default `false`)
       --https_client_timeout int                 Amount (in seconds) of time the https client has before timing out requests. (default `60`)
       --upload_region                            The region the metrics-agent will upload data to. (default `us-west-2`)
-      --custom_s3_bucket string                  A custom S3 bucket the metrics-agent will upload data to. - Optional
-      --custom_s3_region                         The AWS region that the custom s3 bucket is created.
 Global Flags:
       --log_format string   Format for log output (JSON,PLAIN) (default "PLAIN")
       --log_level string    Log level to run the agent at (INFO,WARN,DEBUG) (default "INFO")
 ```
 
+## Deployment of Metrics-Agent
+
+There are two ways to deploy Metrics-agent:
+
+### Deployment using yaml
+
+Cloudability customers can download deployment yaml directly from Cloudability UI. The downloaded yaml contains default settings including API key configured to enable metrics-agent to upload metrics to Cloudability. The customer should change the default settings in the yaml according to their clusters' configuration and security requirements.
```

Comment:
couple minor nits:

- customers can download **_the_** deployment yaml
- contains default settings including **_the_** API key **_needed_** to enable **_the_** metrics-agent to upload to Cloudability.
- **_The_** API key is currently configured as **_an_** **_environment_** variable in the pod as plain text.
- can use helm to deploy **_the_** metrics-agent